### DISCUSSION TITLE
v0.10.1 🎉

### DISCUSSION THREAD

**Discussion by tnir** [n-ronn/nronn#45] [Announcements]
# tl;dr
```sh
$ gem install nronn
```
You should `gem uninstall ronn` or `gem uninstall ronn-ng` if you already have either.
Or
```sh
$ bundle remove ronn ronn-ng || true
$ bundle add nronn
```

### Exciting New Features 🎉
* Ruby 3.2-dev, 3.1 support as well as 2.5, 2.4 support in addition to the original Ruby 2.6, 2.7, 3.0 support.
* Switch to GitHub Flavored Markdown, fixing code block rendering in https://github.com/apjanke/ronn-ng/pull/53
* Bump to mustache 1.x in https://github.com/apjanke/ronn-ng/pull/46
* Remove non-portable `more -i` option in https://github.com/apjanke/ronn-ng/issues/71
* Introduce nRonn 0.10.1.pre2 by @tnir in https://github.com/n-ronn/nronn/pull/3
### Important Bugfixes 🚧
* Revert `\[ci]` back to `\(bu)` in https://github.com/apjanke/ronn-ng/pull/51
* Minor fix to single-quote escaping in https://github.com/apjanke/ronn-ng/issues/55
* Elide HTML comments when producing roff output in https://github.com/apjanke/ronn-ng/issues/65
* Replace CI with GitHub Actions by @tnir in https://github.com/n-ronn/nronn/pull/8
* Relax nokogiri dependency from 1.11.0 to 1.10.10 by @tnir in https://github.com/n-ronn/nronn/pull/10
* Workaround for psych 4.0 in tests by @tnir in https://github.com/n-ronn/nronn/pull/15
* Update links to man pages by @tnir in `https://github.com/n-ronn/nronn/pull/17`
* Update GitHub Pages domain by @tnir in https://github.com/n-ronn/nronn/pull/18
* Check if tests pass with Ruby 2.4-2.6 by @tnir in https://github.com/n-ronn/nronn/pull/19
* Fix dead links happened recently by @tnir in https://github.com/n-ronn/nronn/pull/22
* Use valid charset in Content-Type in HTML output by @tnir in https://github.com/n-ronn/nronn/pull/27
* Fix bugs in rake rev by @tnir in https://github.com/n-ronn/nronn/pull/38
* Improve comments in rake rev by @tnir in https://github.com/n-ronn/nronn/pull/39
### Document📝 
* Update man pages by rake man by @tnir in https://github.com/n-ronn/nronn/pull/11
* Enable GitHub Pages by @tnir in https://github.com/n-ronn/nronn/pull/13
* Use GitHub releases feature as primary changelog by @tnir in https://github.com/n-ronn/nronn/pull/30
* Add tnir to AUTHORS by @tnir in https://github.com/n-ronn/nronn/pull/31
* Add badges on README by @tnir in https://github.com/n-ronn/nronn/pull/37
* Improve gem description by @tnir in https://github.com/n-ronn/nronn/pull/41
### Other Changes
* Release v0.10.1.pre3 by @tnir in https://github.com/n-ronn/nronn/pull/12
* Bump rubocop from 0.93.1 to 1.12.1 by @tnir in https://github.com/n-ronn/nronn/pull/21
* Use main as the default branch by @tnir in https://github.com/n-ronn/nronn/pull/32
* Enable Gemspec/DateAssignment cop by @tnir in https://github.com/n-ronn/nronn/pull/33
* Update Bundler used in bundle install by @tnir in https://github.com/n-ronn/nronn/pull/34
* Remove test files from gem by @tnir in https://github.com/n-ronn/nronn/pull/36
* Release nronn 0.10.1.pre4 by @tnir in https://github.com/n-ronn/nronn/pull/40
* Release nronn 0.10.1 by @tnir in https://github.com/n-ronn/nronn/pull/42
* Add release.yml by @tnir in https://github.com/n-ronn/nronn/pull/44

## New Contributors
* @tnir made their first contribution in https://github.com/n-ronn/nronn/pull/3

**Full Changelog**: https://github.com/n-ronn/nronn/compare/v0.9.1...v0.10.1

<hr /><em>This discussion was created from the release <a href='https://github.com/n-ronn/nronn/releases/tag/v0.10.1'>v0.10.1</a>.</em>
